docs/index.rst

```diff
@@ -3,17 +3,17 @@
 Welcome to GlobalFoundries 0.18UM 3.3V/(5V)6V MCU PDK's documentation!
 ****************************************************************************
 
-.. include:: common.inc
```

Comment:
I don't think this remove is needed any more?

Reply:
Duplicate thus produce warning.